src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -166,6 +166,21 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="..\GitHub.TeamFoundation.14\obj\$(Configuration)\GitHub.TeamFoundation.14.dll">
+      <Link>GitHub.TeamFoundation.14.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+    <Content Include="..\GitHub.TeamFoundation.15\obj\$(Configuration)\GitHub.TeamFoundation.15.dll">
+      <Link>GitHub.TeamFoundation.15.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+    <Content Include="..\GitHub.TeamFoundation.16\obj\$(Configuration)\GitHub.TeamFoundation.16.dll">
+      <Link>GitHub.TeamFoundation.16.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <IncludeInVSIX>true</IncludeInVSIX>
```

Comment:
Keep these as ProjectReference items, but add `ReferenceOutputAssembly=false` metadata to each one. That *should* stop MSBuild from complaining that the target framework of the referenced project exceeds that of this project.

Reply:
Thanks for the suggestion! Unfortunately Visual Studio 2019 still shows errors, even though the build is actually completing.

![image](https://user-images.githubusercontent.com/11719160/52271863-cfa53c80-293c-11e9-99aa-0ccbc2674684.png)

I wonder if we should refactor the solution so that there is a project dedicated to packaging the VSIX that doesn't contain any actual code? This could target .NET 4.7.2 and therefore reference projects that target any other version.

Reply:
> I wonder if we should refactor the solution so that there is a project dedicated to packaging the VSIX that doesn't contain any actual code? This could target .NET 4.7.2 and therefore reference projects that target any other version.

I think _that's_ a great idea. I typically prefer such a pattern anyway so that packing doesn't have to happen to unit test that code.

Regrading VS error list entries, that *might* be tracked by this: https://github.com/Microsoft/msbuild/issues/2661. Can you review that issue and see if it covers the gap? There are workarounds disclosed there as well.

Reply:
This is how I like to structure my personal VSIX projects as well. I've put off doing this with `GitHub.VisualStudio` because it contains quite a lot of code and dependencies, but this build error might be just the incentive we need! 🤞